test/fprime/fbuild/test_settings.py

```diff
@@ -39,6 +39,7 @@ def test_settings():
                 "library_locations": [],
                 "environment_file": full_path("settings-data/settings-empty.ini"),
                 "environment": {},
+                "cookiecutter": "native",
```

Comment:
These probably need to changed from native to default to match with code changes.
Also, adding a test case for explicitly setting the cookiecutter template would be awesome.

Reply:
Ok! I made this test case, but I get the same error with or without my test case when running pytest: that the code can't find the fprime root.

Reply:
Yeah, you have to set the python root before running tests. Try `export PYTHONPATH=$PWD/src` when you're in the repository.